dumps.Makefile

```diff
@@ -88,34 +104,51 @@ $(FINAL_DUMPS_DIR)/solr.json: $(FINAL_DUMPS_DIR)/obographs.json $(RAW_DUMPS_DIR)
 # 1. pick name, add to the correct DUMPS variable (DUMPS_SOLR, DUMPS_PDB, DUMPS_OWLERY)
 # 2. create new sparql query in sparql/, naming it 'construct_name.sparql', e.g. sparql/construct_image_names.sparql
 # Note that non-sparql goals, like 'inferred_annotation', need to be added separately
+
+# Specifies the names of the dumps used to generate the SOLR index.
 DUMPS_SOLR=all preferred_roots deprecation_label image_names has_image
+# Specifies the names of the dumps used to generate the PDB.
 DUMPS_PDB=all preferred_roots deprecation_label has_image
+# Specifies the names of the dumps used to generate the OWLery.
 DUMPS_OWLERY=all
+
+# Specifies the location where the CSV import files are stored.
 CSV_IMPORTS="$(FINAL_DUMPS_DIR)/csv_imports"
+
+# Specifies the JAR file used to convert OWL files to CSV format for import into Neo4j.
 OWL2NEOCSV="$(SCRIPTS_DIR)/owl2neo4jcsv.jar"
 
+# Creates the CSV_IMPORTS directory.
 $(CSV_IMPORTS):
 	mkdir -p $@
 
+# Generates the obographs.json file, which is used to generate the SOLR index.
 $(FINAL_DUMPS_DIR)/obographs.json: $(patsubst %, $(RAW_DUMPS_DIR)/construct_%.owl, $(DUMPS_SOLR)) $(RAW_DUMPS_DIR)/inferred_annotation.owl $(RAW_DUMPS_DIR)/unique_facets.owl
 	echo $@ started: `date +%s` >> $(LOG_FILE)
 	$(ROBOT) merge $(patsubst %, -i %, $^) convert -f json -o $@ $(STDOUT_FILTER)
 	echo $@ ended: `date +%s` >> $(LOG_FILE)
 
+# Generates the PDB.owl file, which is used to generate the PDB.
 PDB_EXTERNAL_ONTS=connectome_fafb.owl connectome_l1em.owl connectome_hemibrain.owl
 $(FINAL_DUMPS_DIR)/pdb.owl: $(patsubst %, $(RAW_DUMPS_DIR)/construct_%.owl, $(DUMPS_PDB)) $(RAW_DUMPS_DIR)/inferred_annotation.owl $(RAW_DUMPS_DIR)/unique_facets.owl $(patsubst %, $(RAW_DUMPS_DIR)/%, $(PDB_EXTERNAL_ONTS))
 	echo $@ started: `date +%s` >> $(LOG_FILE)
 	$(ROBOT) -vvv merge $(patsubst %, -i %, $^) -o $@ $(STDOUT_FILTER)
 	echo $@ ended: `date +%s` >> $(LOG_FILE)
 
+# Generates the owlery.owl file, which is used for other purposes.
 $(FINAL_DUMPS_DIR)/owlery.owl: $(patsubst %, $(RAW_DUMPS_DIR)/construct_%.owl, $(DUMPS_OWLERY))
 	echo $@ started: `date +%s` >> $(LOG_FILE)
 	$(ROBOT) filter -i $< --axioms "logical" --preserve-structure true annotate --ontology-iri "http://virtualflybrain.org/data/VFB/OWL/owlery.owl" -o $@ $(STDOUT_FILTER)
 	echo $@ ended: `date +%s` >> $(LOG_FILE)
 
+# Generates the CSV files for the PDB and imports them into Neo4j.
 pdb_csvs: $(FINAL_DUMPS_DIR)/pdb.owl | $(CSV_IMPORTS)
```

Comment:
there are two `pdb_csvs` goals, this can be a merge error. It should be something like this:
```
pdb_csvs: $(FINAL_DUMPS_DIR)/pdb.owl | $(CSV_IMPORTS)
	echo $@ started: `date +%s` >> $(LOG_FILE)
	java $(ROBOT_ARGS) -jar $(OWL2NEOCSV) $< "$(VFB_CONFIG)" $(CSV_IMPORTS) false $(INFER_ANNOTATE_RELATION)
	echo $@ ended: `date +%s` >> $(LOG_FILE)
	echo "=== Print Timer Logs ==="
	echo "`cat $(LOG_FILE)`"
```